packages/uniforms/src/SimpleSchemaBridge.js

```diff
@@ -202,8 +202,8 @@ export default class SimpleSchemaBridge extends Bridge {
             } else {
                 field = {
                     ...field,
-                    transform: value => options.find(option => option.value === value).label,
-                    allowedValues: options.map(option => option.value)
+                    transform: value => options.find(option => option.value.toString() === value).label,
+                    allowedValues: options.map(option => option.value.toString())
```

Comment:
I had a issue with my select only returning values and not labels. I found that the options I was passing had some of the labels undefined.. So I would recommend carefully checking the results of `PlaceTypeEnum.getOptions()` in you schema there. But also, I had problems when one of the items in the array was `null` but can't remember if it was related.